src/fontra/client/core/utils.js

```diff
@@ -353,6 +353,44 @@ export function rgbaToCSS(rgba) {
   return `rgb(${channels.join(",")})`;
 }
 
+export function hexToRgba(hexColor) {
+  let c = hexColor.substring(1).split("");
+  let r = [];
+  if (/^#[A-Fa-f0-9]{8}$/.test(hexColor) || /^#[A-Fa-f0-9]{6}$/.test(hexColor)) {
+    for (const i of range(0, c.length, 2)) {
+      r.push(round(parseInt(c[i] + c[i + 1], 16) / 255, 2));
```

Comment:
2 digits is not enough, you need three to be able to reliably round-trip.

I would go for 4, though, for some added precision.

Please also test round tripping between hexToRgba and rgbaToHex.

Reply:
done.